sound/smooth_swing_v2.h

```diff
@@ -148,6 +166,12 @@ class SmoothSwingV2 : public SaberBasePassThrough {
 	    // We need to stop setting the volume when off, or playback may never stop.
 	    A.set_volume(mixhum * mixab);
 	    B.set_volume(mixhum * (1.0 - mixab));
+      if (delegate_->IsSwingPlaying()) {
```

Comment:
Let's call IsSwingPlaying inside SetSwingVolume instead.
Then this class will not need to call IsSwingPlaying and it can be removed from SaberBase.